dev-aws/kafka-shared/iam-customer.tf

```diff
@@ -0,0 +1,35 @@
+resource "kafka_topic" "iam_customer_v1" {
+  name               = "iam-customer-v1"
+  replication_factor = 3
+  partitions         = 10
+  config = {
+    # retain 100MB on each partition
+    "retention.bytes" = "104857600"
+    # keep data for 2 days
+    "retention.ms" = "172800000"
```

Comment:
2 days because of dev?

Reply:
Yeah but might be a bit aggressive. remove it or make it 90d?

Reply:
Yeah, I'd probably keep at least 7d worth of events even in dev.